include/mbedtls/pk.h

```diff
@@ -535,6 +535,52 @@ int mbedtls_pk_sign( mbedtls_pk_context *ctx, mbedtls_md_type_t md_alg,
              unsigned char *sig, size_t sig_size, size_t *sig_len,
              int (*f_rng)(void *, unsigned char *, size_t), void *p_rng );
 
+#if defined(MBEDTLS_USE_PSA_CRYPTO)
```

Comment:
Suggestion: make the function available unconditionally, but document that it only support `pk_type == MBEDTLS_PKRSASSA_PSS` if `PSA_CRYPTO_C` is defined.